llama_index/llms/huggingface.py

```diff
@@ -294,3 +306,135 @@ def stream_chat(
         prompt = self._messages_to_prompt(messages)
         completion_response = self.stream_complete(prompt, formatted=True, **kwargs)
         return stream_completion_response_to_chat_response(completion_response)
+
+
+def conversational_output_to_chat_response(
+    output: "ConversationalOutput", role: MessageRole = MessageRole.ASSISTANT
+) -> ChatResponse:
+    return ChatResponse(message=ChatMessage(role=role, content=output.generated_text))
+
+
+class HuggingFaceInferenceAPI(LLM):
```

Comment:
nit: would prefer for this to live in a separate python file.

Reply:
It's named `huggingface`, so I thought it's fine to live in the same one.  What do you think of me renaming the two modules to:
- `hf_transformers`
- `hf_inference_api`

Reply:
Fwiw, the Python builtin libraries have many classes inside, I think it can also make sense to keep all Hugging Face classes inside one `huggingface` module